src/roles/nuh-deploy/tasks/nuh_copy_certificates.yml

```diff
@@ -25,6 +24,10 @@
         additional_parameters:  -d {{ item.external_fqdn }}
       with_items:
         - "{{ external_interfaces }}"
+      when: 
```

Comment:
If you don't check to see if the external_interfaces are defined, does this particular task fail? Or does it just get skipped?

Reply:
I believe we added that because it was failing without it. I'll try it one more time to be sure.

Reply:
yeah let's see what happens if we don't check if the external_interfaces are defined

Reply:
Tried it. Not needed. Taking it out.

Reply:
ok cool